src/button/default.scss

```diff
@@ -37,15 +37,20 @@
     }
   }
 
-  &.iui-active {
+  &.iui-active:enabled {
+    position: relative;
     @include themed {
-      background: linear-gradient(
-        rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6)),
-        rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6))
-      ),
-        linear-gradient(t(iui-color-background-1), t(iui-color-background-1));
       color: t(iui-color-foreground-primary);
     }
+
+    &::before {
+      content: '';
+      position: absolute;
+      inset: 0;
```

Comment:
![image](https://user-images.githubusercontent.com/36186912/149723319-5df511fc-7262-4d3d-85f4-9d96cb2b02e8.png)
So only the blue stripe is showing that it is active which is barely visible

Reply:
I can't see icons in IE. 
If we are dropping IE support, this is not a problem.
![image](https://user-images.githubusercontent.com/83585998/149770700-2464fb17-851d-47bc-b3ed-d238e2893d67.png)

Reply:
> blue stripe is showing that it is active which is barely visible

I would not call it "barely visible". It is functional and that's good enough for progressive enhancement

>I can't see icons in IE. 

That's because we are using webcomponents in demo pages. The CSS itself is fine

Reply:
So what is stalling here? Is the fix regression on IE?
Could we wait for official drop time and only then start fixing things which break on IE?
What are other concerns, because this PR is staying here for a very long time. (css selector specificity increased...?)